core/internal/gqlmock/matchers.go

```diff
@@ -63,8 +64,17 @@ func (m *queryVariablesMatcher) Matches(x any) bool {
 		return false
 	}
 
+	// Match values at nested paths, if needed
```

Comment:
could you open a new PR for this change?

Reply:
let's just quickly merge this change and follow up, we merge this PR

Reply:
got it will just merge this PR for now then

Reply:
never mind you merged it in the wrong order! but fine, in the future let's try to avoid these huge PRs if we can

Reply:
my mistake -- I misunderstood the suggestion/ask as merging this PR first.  Agreed about avoiding huge PRs, will be more mindful going forward